apis/controller/v1alpha1/sliceconfig_types.go

```diff
@@ -69,12 +78,11 @@ type SliceConfigSpec struct {
 
 // ExternalGatewayConfig is the configuration for external gateways like 'istio', etc/
 type ExternalGatewayConfig struct {
-	Ingress   ExternalGatewayConfigOptions `json:"ingress,omitempty"`
-	Egress    ExternalGatewayConfigOptions `json:"egress,omitempty"`
-	NsIngress ExternalGatewayConfigOptions `json:"nsIngress,omitempty"`
-	//+kubebuilder:validation:Enum:=none;istio
-	GatewayType string   `json:"gatewayType,omitempty"`
-	Clusters    []string `json:"clusters,omitempty"`
+	Ingress     ExternalGatewayConfigOptions `json:"ingress,omitempty"`
+	Egress      ExternalGatewayConfigOptions `json:"egress,omitempty"`
+	NsIngress   ExternalGatewayConfigOptions `json:"nsIngress,omitempty"`
+	GatewayType GatewayType                  `json:"gatewayType,omitempty"`
```

Comment:
As per the doc, the new field should be named "ServiceAccess", dont see it here. Please add a new section "VPCServiceAccess".

Reply:
addressed